src/include/platform/ConnectivityManager.h

```diff
@@ -245,6 +259,10 @@ class ConnectivityManager
     static const char * CHIPoBLEServiceModeToStr(CHIPoBLEServiceMode mode);
 
 private:
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    WiFiPAFAdvertiseParams mPafAdverParam;
```

Comment:
```suggestion
    WiFiPAFAdvertiseParams mPAFAdvertisementParams;
```
Or even "Parameters"....  Or at least `mPAFAdvertiseParams`.  But not "Adver", please.